code/modules/mob/living/living.dm

```diff
@@ -277,9 +277,9 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] passively!</span>", \
-							"<span class='warning'>[src] grabs you passively!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] passively!</span>")
+			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? " by their hands!":"passively!"]</span>", \
```

Comment:
```suggestion
			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
```

Reply:
please don't suggest single-lining when humans don't read horizontally into infinity.

Reply:
I'm not suggesting that now, am I?

Reply:
Oh, right, PR author suggests that.